tests/Application/config/packages/setono_sylius_image.yaml

```
@@ -7,3 +7,5 @@ setono_sylius_image:
         sylius_shop_product_small_thumbnail: ~
         sylius_shop_product_thumbnail: ~
         sylius_shop_product_large_thumbnail: ~
+    processable_resources: 
```

```suggestion
    resources: 
```

```suggestion
    image_resources: 
```

One of these instead?